src/dbus/server/dbus_thread_object.cpp

```diff
@@ -1163,6 +1168,32 @@ otError DBusThreadObject::GetActiveDatasetTlvsHandler(DBusMessageIter &aIter)
     return error;
 }
 
+otError DBusThreadObject::SetFeatureFlagListDataHandler(DBusMessageIter &aIter)
+{
+    std::vector<uint8_t> data;
+    FeatureFlagList      featureFlagList;
+    otError              error = OT_ERROR_NONE;
+
+    VerifyOrExit(DBusMessageExtractFromVariant(&aIter, data) == OTBR_ERROR_NONE, error = OT_ERROR_INVALID_ARGS);
+    VerifyOrExit(featureFlagList.ParseFromString(std::string(data.begin(), data.end())), error = OT_ERROR_INVALID_ARGS);
+    mLastParsedFeatureFlagListBytes = std::move(data);
+    // TODO: apply the feature flag list to Thread instance.
```

Comment:
What's stopping us from implementing the actual handler in this PR?

Reply:
Maybe we can declare the ApplyFeatureFlagList() function but leave it not implemented?

Reply:
Added initial ApplyFeatureFlagList now.